### ISSUE TITLE
Access to the dataset

### ISSUE THREAD

**Issue by ianblair87** [activitynet/ActivityNet#104]
Hello. Is there any update on accessibility of the dataset? It seems like this has been unresolved for some time now, but fiftyone is unable to load the dataset, since the annotations link is non-functional. Thanks

**Comment by escorciav** [comment]
#103
